docs/app/Components/ComponentDoc/ComponentDescription.js

```diff
@@ -1,68 +1,104 @@
 import _ from 'lodash'
 import React, { Component, PropTypes } from 'react'
-import SEMANTIC_TYPES from 'docs/app/utils/SemanticTypes'
-import STARDUST_TYPES from 'docs/app/utils/StardustTypes'
-import { Icon, Segment } from 'stardust'
+import { Divider, Grid, Header, List } from 'stardust'
+import META from 'src/utils/Meta'
 
-export default class ComponentDescription extends Component {
-  static propTypes = {
-    /**
-     * String describing the Stardust component.
-     */
-    description: PropTypes.string.isRequired,
+const headerStyle = {
+  fontSize: '2.2em',
+}
 
-    /**
-     * Name of the Stardust component
-     */
-    name: PropTypes.string.isRequired,
+const descriptionStyle = {
+  fontSize: '1.2em',
+}
 
-    /**
-     * Name of the Stardust parent component (i.e. 'Grid' for Column).
-     */
-    parent: PropTypes.string.isRequired,
+export default class ComponentDescription extends Component {
```

Comment:
This component still renders the top section of a component's documentation:

![image](https://cloud.githubusercontent.com/assets/5067638/14833802/39fdb444-0bbe-11e6-9c6d-51af126e1896.png)